apiml-common/src/main/java/com/ca/mfaas/product/web/HttpConfig.java

```diff
@@ -96,6 +99,13 @@ public void init() {
             secureHostnameVerifier = factory.createHostnameVerifier();
             eurekaJerseyClientBuilder = factory.createEurekaJerseyClientBuilder(eurekaServerUrl, serviceId);
 
+            HttpsConfig httpsConfigWithoutKeystore = HttpsConfig.builder().protocol(protocol).trustStore(trustStore)
```

Comment:
What about use lambda:

            Supplier<HttpsConfig.HttpsConfigBuilder> httpsConfigBuilderSupplier = () ->
                HttpsConfig.builder().protocol(protocol)
                .trustStore(trustStore)
                .trustStoreType(trustStoreType).trustStorePassword(trustStorePassword).trustStoreRequired(trustStoreRequired)
                .verifySslCertificatesOfServices(verifySslCertificatesOfServices);


HttpsFactory factory = new HttpsFactory(httpsConfig);

HttpsFactory factoryWithoutKeyStore = new HttpsFactory(httpsConfigBuilderSupplier.get().build());
            secureHttpClientWithoutKeystore = factoryWithoutKeyStore.createSecureHttpClient();

Reply:
That will look nicer! I have created an issue so we do not forget about